<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ninad-Legion" failures="1" tests="3" name="DemoTest" time="15.914" errors="0" timestamp="2023-10-11T20:53:18 IST">
  <testcase classname="demo.AddToCartTest" name="invalidLoginTest" time="2.382">
    <failure type="junit.framework.AssertionFailedError" message="Test Failed as ProductCatalog Object returned by login in Not Null">
      <![CDATA[junit.framework.AssertionFailedError: Test Failed as ProductCatalog Object returned by login in Not Null
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertNull(Assert.java:233)
at demo.AddToCartTest.invalidLoginTest(AddToCartTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginTest -->
  <testcase classname="demo.AddToCartTest" name="loginTest" time="2.441"/>
  <testcase classname="demo.AddToCartTest" name="addProductToCartTest" time="8.753"/>
</testsuite> <!-- DemoTest -->
